src/execution/operator/aggregate/physical_streaming_window.cpp

```diff
@@ -50,41 +50,60 @@ class StreamingWindowState : public OperatorState {
 public:
 	using StateBuffer = vector<data_t>;
 
-	StreamingWindowState()
-	    : initialized(false), allocator(Allocator::DefaultAllocator()),
-	      statev(LogicalType::POINTER, data_ptr_cast(&state_ptr)) {
-	}
+	struct AggregateState {
```

Comment:
Won't this name conflict with `struct AggregateState` in `physical_ungrouped_aggregate.cpp` in our amalgamation builds?

Reply:
I don't think so - it is an inner class.